config/webpack/environment.js

```diff
@@ -40,6 +40,12 @@ $image-path: '~identity-style-guide/dist/assets/img';
 @import '~identity-style-guide/dist/assets/scss/uswds/utilities/rules/all';
 @import '~identity-style-guide/dist/assets/scss/uswds/utilities/rules/package';`;
 
+sassLoader.use.map((loader) => {
+  if (loader.loader === "css-loader") {
+    loader.options.sourceMap = false;
+  }
+});
+
```

Comment:
`forEach` would technically be more appropriate here, since we don't care to do anything with the returned value of `map`.

Alternatively, if we're assuming we should have exactly one instance of a `css-loader` in the array (which I believe is the case), then we could simplify this a bit further, potentially save a couple iterations, and be consistent with what we're doing just above with `sass-loader`:

```suggestion
sassLoader.use.find(({ loader }) => loader === 'css-loader').options.sourceMap = false;

```